provd/proto/accessibility.proto

```diff
@@ -0,0 +1,87 @@
+syntax = "proto3";
+
+package accessibility;
+
+service AccessibilityDirectoryService {
```

Comment:
I am assuming we would like these toggles to take immediate effect? So I have split them into individual requests and responses so we can update them via the screen and a user can test drive them.